nrich-notification/README.md

```diff
@@ -33,33 +46,76 @@ To be able to use this module following configuration is required:
     public NotificationResponseService notificationResponseService(NotificationResolverService notificationResolverService) {
         return new WebMvcNotificationResponseService(notificationResolverService);
     }
-}
 
+}
 ```
 
-`ConstraintConversionService` is a service responsible for converting `javax.validation.ConstraintViolation` list to Spring `Errors`
-instances making implementation of `NotificationResolverService` less complex since it only needs to work with Spring's `Errors` when processing validation errors.
+### [ConstraintConversionService][constraint-conversion-service-url]
```

Comment:
Please take a look at the second comment in https://github.com/croz-ltd/nrich/pull/121#discussion_r933837027. If we agree to change it there, we should do it here.

Reply:
Yep, I changed it.